OpenVBX/views/messages/details.php

```diff
@@ -124,6 +124,43 @@
 			<h3>Transcription</h3>
 			<?php endif; ?>
 			<div class="message-transcript"><?php echo (is_null($message->content_text) ? "(no transcription)" : $message->content_text) ?></div>
+			<?php if(substr($message->call_sid,0,2) == 'MM'):
+				echo "<br><br><h3>Media Attachements</h3>";
+				
+				$ci = & get_instance();
+				
+				require_once(APPPATH . 'libraries/twilio.php');
```

Comment:
This library is old and deprecated. It should not be used. Instead use the newer `Services/Twilio` php library. As mentioned above this code should be moved to the controller and the media information added to the `$details` array. Add something like this between lines 354 & 356 in `controllers/messages/details.php`:

```php
if (substr($message->call_sid,0,2) == 'MM') {
	$account = OpenVBX::getAccount();
	$message = $account->messages->get($message->call_sid);

	foreach ($message->media as $mediaItem) {
		$details['media'][] = array(
			'sid' => $mediaItem->sid, 
			'content_type' => $mediaItem->content_type, 
			'uri' => $mediaItem->uri,
		);
	}
}
```